### DISCUSSION TITLE
Elastalert rule location

### DISCUSSION THREAD

**Discussion by PatRussell87** [Security-Onion-Solutions/securityonion#6117] [Unsupported Versions] [answered]
Wondering where to find the final resting place for Elastalert/Playbook rules. I have several place in /opt/so/rules/elastalert/playbook but have accidentally overwritten one and am hoping to recover it. I believe there is another copy in use somewhere since the alert is still working as if it had not been changed.

Is there a way to revert (no...I did not make a copy first, lesson learned) or copy the in-use file back?

**Comment by dougburks** [comment, marked as answer]
You can try deleting all the *.yaml files under `/opt/so/rules/elastalert/playbook/` and then running `so-playbook-sync` to regenerate the files:
https://docs.securityonion.net/en/2.3/playbook.html#misc-notes

**Reply by PatRussell87** [reply]
Thanks Doug. I'll remember that next time. It was only one rule this time so I just recreated it.
